Procfile

```diff
@@ -0,0 +1,3 @@
+default_process_types:
+  web: ./dub-registry --port $PORT --mirror=https://code.dlang.org --hostname=dub-registry.heroku.com --separate-cron --vv --bind 0.0.0.0
+  cron: ./dub-registry --mirror=https://code.dlang.org --run-cron --vv
```

Comment:
Not particularly nice as a cron task in a separate process cannot update in-process caches.
Might be problematic during future development, just support a minor Heroku suspend case.
Any other ideas?

Reply:
Not really. Heroku instances fall asleep very soon, so chances are good that the main dyno gets suspended / terminated a lot anyways.